chirp/drivers/vgc.py

```diff
@@ -549,7 +549,7 @@ def _upload(radio):
 
 def model_match(cls, data):
     """Match the opened/downloaded image to the correct version"""
-    rid = data[0x6140:0x6148]
+    rid = data[0x6140:0x6148].decode('ascii', 'replace')
```

Comment:
I think the right way to do this is encode the model string, which we know will encode to UTF-8, instead of decode-and-discard unknown binary data. I'll push a change.